index/index.go

```diff
@@ -36,7 +36,6 @@ const (
 	// MagicIndex 4 bytes at the head of an index file.
 	MagicIndex = 0xBAAAD700
 
-	indexFormatV1 = 1
```

Comment:
btw this shouldn't be removed , but refactored like in

https://github.com/prometheus/tsdb/pull/492/files#diff-71ebe2bcf31a915b1fa3b3b289d5d31dR651